hack/snapshot_to_catalog.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
+
+usage() {
+  echo "Usage: $0 <snapshot-ref> <catalog-file> <bundle-tag>"
+  echo "  snapshot-ref: the snapshot reference to use, example: ols-bnxm2"
+  echo "  catalog-file: the catalog index file to update, example: lightspeed-catalog-4.16/index.yaml"
+  echo "  bundle-tag: the tag of the bundle to update, example: 0.1.0"
+  exit 1
+}
+
+if [ $# == 0 ]; then
+  usage
+fi
+
+SNAPSHOT_REF=${1:-"ols-bnxm2"}
+CATALOG_FILE=${2:-"lightspeed-catalog-4.16/index.yaml"}
+BUNDLE_TAG=${3:-"0.1.0"}
+echo "update bundle ${BUNDLE_TAG} catalog ${CATALOG_FILE} from snapshot ${SNAPSHOT_REF}"
+
+# cache the snapshot from Konflux
+TMP_SNAPSHOT_JSON=$(mktemp)
```

Comment:
in following code it is better to double quote it (in all places)

Reply:
all quoted ;)